sci-astronomy/celestia/celestia-1.6.2.2.recipe

```diff
@@ -11,8 +11,8 @@ simple to navigate through the universe to the object you want to visit.
 Celestia is expandable. Celestia comes with a large catalog of stars, galaxies, \
 planets, moons, asteroids, comets, and spacecraft. If that's not enough, you \
 can download dozens of easy to install add-ons with more objects."
-HOMEPAGE="http://celestia.sourceforge.net"
-COPYRIGHT="2001-2020 Celestia Development Team"
+HOMEPAGE="https://celestia.space"
+COPYRIGHT="2001-2021 Celestia Development Team"
 LICENSE="GNU GPL v2"
 REVISION="1"
```

Comment:
Change this to `2`.

Reply:
1.6.2.2 was released in 2021 so it shouldnt be 2022(?)